ai-platform-unified/notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -1,820 +1,1037 @@
 {
```

Comment:
<p>This simple example didn't use any of the the Vertex AI training/serving services, create a dataset &amp; endpoint, etc. So this will probably confuse those who are paying attention :)  -- I suggest it (and the the code below) be pared down to just what would need to be deleted.  </p><p> </p><p>Also -- and this probably holds for all the notebooks-- I'm not sure that it's necessary to delete the pipeline run info.  After it has finished the run, I don't believe that the record of the run would incur any charges? (though we could double check). The user might want to retain the run info for reference.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>I know its an overkill, but its an artifact of using auto-assembly -- clean any and all. I will look into updating the text to avoid confusion.</p>